src/minidump.rs

```diff
@@ -889,6 +889,13 @@ impl<'a> MinidumpMemory<'a> {
         desc: &md::MINIDUMP_MEMORY_DESCRIPTOR,
         data: &'a [u8],
     ) -> Result<MinidumpMemory<'a>, Error> {
+        if desc.memory.rva == 0 || desc.memory.data_size == 0 {
```

Comment:
The `location_slice` function called below should probably be fixed to handle this. It's my "turn an RVA into a byte slice" utility function, so fixing it there would fix this for most places that want to use an RVA like this.

Reply:
I originally did that but it made `test_crashpad_info` explode. There's a few places where `rva=0` appears to be legit, but never for MinidumpMemory specifically.